scripts/build.js

```diff
@@ -27,6 +28,7 @@ if (glob) {
   if (extraArgs.includes('--lint')) {
     run(`nimbus eslint {packages,plugins}/${glob}/{src,test}`);
   }
+  rimraf.sync(`./{packages,plugins}/${glob}/{lib,esm,tsconfig.tsbuildinfo}`);
```

Comment:
Not super relevant here, but this reminds me, it might be nice to do something similar to remove package-lock.json files that sometimes sneak their way into plugins, which break the `yarn ci:create-patch-version` process.

Reply:
`pckage-lock.json` doesn't seem to matter for the build process. I'll create a PR to remove them before `ci:create-patch-version`.